signer/local/local.go

```diff
@@ -254,7 +259,66 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
-	return s.sign(&safeTemplate, profile)
+	var certTBS = safeTemplate
+
+	if len(profile.CTLogServers) > 0 {
+		// Add a poison extension which prevents validation
+		var oidCTPoison = []int{1, 3, 6, 1, 4, 1, 11129, 2, 4, 3}
+		var poisonExtension = pkix.Extension{Id: oidCTPoison, Critical: true, Value: []byte{0x05, 0x00}}
+		var poisonedPreCert = certTBS
+		poisonedPreCert.ExtraExtensions = append(safeTemplate.ExtraExtensions, poisonExtension)
+		cert, err = s.sign(&poisonedPreCert, profile)
+
+		if err != nil {
+			return
+		}
+
+		derCert, _ := pem.Decode(cert)
+		prechain := []ct.ASN1Cert{derCert.Bytes, s.ca.Raw}
+
+		var sctList []ct.SignedCertificateTimestamp;
+
+		for _, server := range profile.CTLogServers {
+			var ctclient = client.New(server)
+			var resp, err = ctclient.AddPreChain(prechain)
+			if err != nil {
+				return nil, cferr.Wrap(cferr.CTError, cferr.Unknown, err)
+			}
+			log.Info(resp)
```

Comment:
It would be useful to add some additional information to this log line: what's being logged and why. Perhaps only log some information, or use formatting to show what the fields are. Otherwise, this should be a `Debug` level log, and shouldn't be shown to users normally.